actions/git-blame-ignore-revs/plugin.yaml

```diff
@@ -0,0 +1,10 @@
+version: 0.1
+actions:
+  definitions:
+    - id: git-blame-ignore-revs
+      display_name: Auto-enable .git-blame-ignore-revs
+      description: Automatically configure git to use .git-blame-ignore-revs while the file exists.
+      run: ${cwd}/update_config.sh
+      triggers:
+        - files: [.git-blame-ignore-revs]
+        - schedule: 24h
```

Comment:
Why the schedule?

Reply:
Just to cause it to run on startup. We need a `startup` trigger.